job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/wrapper/WrapperContext.groovy

```diff
@@ -661,4 +661,24 @@ class WrapperContext extends AbstractExtensibleContext {
             jiraFilter(context.filter ?: '')
         }
     }
+
+    @RequiresPlugin(id = 'versionnumber')
+    void versionNumber(@DslContext(VersionNumberContext) Closure closure = null) {
+        VersionNumberContext context = new VersionNumberContext()
+        ContextHelper.executeInContext(closure, context)
+
+        wrapperNodes << new NodeBuilder().'org.jvnet.hudson.tools.versionnumber.VersionNumberBuilder' {
+            versionNumberString(context.versionNumberString ?: '')
+            projectStartDate(context.projectStartDate ?: new Date())
+            environmentVariableName(context.environmentVariableName ?: 'VERSION')
```

Comment:
as far as I see VERSION is not a default value with an empty version number build wrapper:
```xml
<org.jvnet.hudson.tools.versionnumber.VersionNumberBuilder plugin="versionnumber@1.6">
<versionNumberString/>
<projectStartDate>1970-01-01 00:00:00.0 UTC</projectStartDate>
<environmentVariableName/>
<environmentPrefixVariable/>
<oBuildsToday/>
<oBuildsThisWeek/>
<oBuildsThisMonth/>
<oBuildsThisYear/>
<oBuildsAllTime/>
<skipFailedBuilds>false</skipFailedBuilds>
<useAsBuildDisplayName>false</useAsBuildDisplayName>
</org.jvnet.hudson.tools.versionnumber.VersionNumberBuilder>
```